### PR TITLE
fastjet: avoid plugins=all,cxx combinations

### PR DESCRIPTION
**Pull request by wdconinc**
Fastjet allows for plugins, including two special keywords `all` and `cxx`. This PR modifies the plugin variant to only allow selection from disjoint sets, so combinations like `all,cxx` or `atlascone,cxx` aren't possible anymore.

### CODE REVIEW

**Review by alalazo — APPROVED** [review]
Waiting for maintainers review. Feel free to merge when it seems appropriate.

**Review by jmcarcell — APPROVED** [review]
Only `PxCone` seems to need fortran.

**Pull request merged**
